composer/algorithms/swa/swa.py

```diff
@@ -30,8 +30,9 @@ class SWA(Algorithm):
     memory required doubles, however, since stored activations and the
     optimizer state are not doubled.
 
-    This algorithm runs on :attr:`~composer.core.event.Event.EPOCH_END` if training
-    duration >= `swa_start`.
+    Uses PyTorch's `torch.optim.swa_util 
```

Comment:
Prefer to use intersphinx cross-reference notation if that works.

```suggestion
    Uses PyTorch's :mod:`torch.optim.swa_util`
```

Reply:
I tried that at first but it doesn't render a link, possibly because `torch.optim.swa_util` isn't actually documented, it just exists in tutorial form.